.github/workflows/test.yml

```diff
@@ -12,6 +12,7 @@ env:
   KIND_CLUSTER_NAME: ovn
   KIND_INSTALL_INGRESS: true
   KIND_ALLOW_SYSTEM_WRITES: true
+  PARALLEL: true
```

Comment:
wait don't you also need to add another run for just serial cases? Otherwise we are missing those right?

Reply:
These are the Serial jobs for sig-network, seems we can skip them, but I have to add a comment explaining it
```
[sig-network] DNS configMap nameserver [IPv4] Forward external name lookup
  should forward externalname lookup to upstream nameserver [Slow][Serial]
  /home/aojea/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/test/e2e/network/dns_configmap.go:515

[sig-network] DNS configMap nameserver [Feature:Networking-IPv6] [LinuxOnly] Forward external name lookup
  should forward externalname lookup to upstream nameserver [Slow][Serial]
  /home/aojea/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/test/e2e/network/dns_configmap.go:550

[sig-network] DNS configMap nameserver [IPv4] Forward PTR lookup
  should forward PTR records lookup to upstream nameserver [Slow][Serial]
  /home/aojea/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/test/e2e/network/dns_configmap.go:506

[sig-network] DNS configMap nameserver [Feature:Networking-IPv6] [LinuxOnly] Forward PTR lookup
  should forward PTR records lookup to upstream nameserver [Slow][Serial]
  /home/aojea/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/test/e2e/network/dns_configmap.go:541

[sig-network] DNS configMap nameserver [IPv4] Change stubDomain
  should be able to change stubDomain configuration [Slow][Serial]
  /home/aojea/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/test/e2e/network/dns_configmap.go:497


[sig-network] DNS configMap nameserver [Feature:Networking-IPv6] [LinuxOnly] Change stubDomain
  should be able to change stubDomain configuration [Slow][Serial]
  /home/aojea/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/test/e2e/network/dns_configmap.go:532


[sig-network] Firewall rule
  [Slow] [Serial] should create valid firewall rules for LoadBalancer type service
  /home/aojea/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/test/e2e/network/firewall.go:71


[sig-network] DNS configMap federations [Feature:Federation]
  should be able to change federation configuration [Slow][Serial]
  /home/aojea/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/test/e2e/network/dns_configmap.go:52

[sig-network] [Feature:PerformanceDNS][Serial]
  Should answer DNS query for maximum number of services per cluster
  /home/aojea/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/test/e2e/network/dns_scale_records.go:55


[sig-network] Loadbalancing: L7 Scalability GCE [Slow] [Serial] [Feature:IngressScale]
  Creating and updating ingresses should happen promptly with small/medium/large amount of ingresses
  /home/aojea/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/test/e2e/network/ingress_scale.go:58

[sig-network] Services
  should reconcile LB health check interval [Slow][Serial]
  /home/aojea/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/test/e2e/network/service.go:2326


````

Reply:
so, I think that mimicing kubernetes upstream strategy, we can run parallel in PR to get quick feedback, and run the serial jobs periodically to check we are not regressing.

Reply:
yeah, we shouldn't be running any of the `[Slow]` tests as part of presubmit CI anyway... setting up a periodical for those would be good.

Reply:
OK I'm good with just parallel for PRs then.